docs/data/material/guides/routing/routing-zh.md

```diff
@@ -16,9 +16,15 @@ You can also make a button perform navigation actions. You can also make a butto
 
 In real-life applications, using a native `<a>` element is rarely enough. element is rarely enough. You can improve the user experience by using an enhanced Link component systematically. The theme of Material UI allows configuring this component once. For instance, with react-router: The theme of MUI allows configuring this component once. For instance, with react-router:
 
-```jsx
+```tsx
```

Comment:
We are not manually changing the translation files. Please revert the changes in the `routing-zh.md` and `routing-pt.md` and we should be good to merge :)

Reply:
@mnajdova  addressed